README.md

```diff
@@ -171,3 +186,7 @@ cd packages/contracts
 pip3 install slither-analyzer
 yarn test:slither
 ```
+
+## License
+
+Files within this repository are licensed under the [MIT License](https://github.com/ethereum-optimism/optimism/blob/master/LICENSE) unless stated otherwise.
```

Comment:
This is not true, `l2geth` is GNU GPLv3 - https://gist.github.com/kn9ts/cbe95340d29fc1aaeaa5dd5c059d2e60
We should be sure to mention that as people who have broken the geth license in the past have caused controversy

Reply:
Will make a note of this

Reply:
Updated